src/main/java/org/prebid/server/spring/config/ServiceConfiguration.java

```diff
@@ -344,8 +344,9 @@ HttpAdapterConnector httpAdapterConnector(HttpClient httpClient, Clock clock) {
 
     @Bean
     RequestValidator requestValidator(BidderCatalog bidderCatalog,
-                                      BidderParamValidator bidderParamValidator) {
-        return new RequestValidator(bidderCatalog, bidderParamValidator);
+                                      BidderParamValidator bidderParamValidator,
+                                      @Value("${blacklisted-apps:#{null}}") String blacklistedApps) {
```

Comment:
From last time we try to avoid in-code defaults. Pls just add empty value to `application.yml`.

Reply:
Was looking for existing `#{null}` cases in our configurations. Maybe we need to review some of them as well if we are moving away from declaring defaults like that?